app/src/androidTest/java/org/mozilla/focus/activity/robots/BrowserRobot.kt

```diff
@@ -241,18 +258,51 @@ class BrowserRobot {
     fun clickOpenLinksInAppsOpenButton() = openLinksInAppsOpenButton.click()
 
     fun clickDropDownForm() {
-        mDevice.findObject(UiSelector().resourceId("$packageName:id/engineView"))
-            .waitForExists(waitingTime)
-        mDevice.findObject(UiSelector().textContains("Drop-down Form")).waitForExists(waitingTime)
-        dropDownForm.click()
+        for (i in 1..RETRY_COUNT) {
+            try {
+                mDevice.findObject(UiSelector().resourceId("$packageName:id/engineView"))
+                    .waitForExists(waitingTime)
+                mDevice.findObject(UiSelector().textContains("Drop-down Form")).waitForExists(waitingTime)
```

Comment:
could this be replaced with dropDownForm.waitForExists instead? If it's necessary to do so than ignore my comment.

Reply:
I'll give it a try, thanks !

Reply:
Done !